web/client/bin/test-lint

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
-
 pushd /opt/irontec/ivozprovider/web/client
     # Install required dependencies
     yarnpkg install
 
     # Run linter
     yarnpkg lint
+
+    CHANGED_FILES=$(git diff --name-only)
```

Comment:
I would add a dot at the end to only check current directory, because the check is run with others in parallel, this may catch differences in files in other paths.

    CHANGED_FILES=$(git diff --name-only .)